stdlib/char.gr

```diff
@@ -315,64 +315,170 @@ provide let (>=) = (x: Char, y: Char) => {
 }
 
 /**
- * Checks if the character is an ASCII digit.
+ * Sub module for working with ASCII characters.
  *
- * @param char: The character to check
- * @returns `true` if the character is an ASCII digit or `false` otherwise
- * 
- * @example assert Char.isAsciiDigit('1')
- * @example assert !Char.isAsciiDigit('a')
+ * @example Char.Ascii.isAscii('1')
  *
- * @since v0.6.0
+ * @since v0.7.0
  */
-provide let isAsciiDigit = char => char >= '0' && char <= '9'
+provide module Ascii {
+  /**
+   * The minimum valid ASCII character code.
+   *
+   * @since v0.7.0
+   */
+  provide let min = 0x00
```

Comment:
Shouldn't these be chars? If someone really needs the code then they can do a `Char.code` on it.

Reply:
We currently use the code on the top level module `Char.min`. I think if we really want to change that we should do it in a seperate pr after this.

Reply:
That makes sense. We can leave it as-is.